src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -345,10 +351,65 @@ def test_0002_bz1209483(self, multihost, backupsssdconf):
         assert "foo2:*:2001:2001::/home/foo2:/bin/bash" \
                in execute_cmd(multihost, "getent passwd -s "
                                          "sss foo2").stdout_text
-        ssh1 = SSHClient(client_e, username="foo2", password="Secret123")
+        ssh1 = SSHClient(client_e, username="foo2",
+                         password="Secret123")
         ssh1.close()
         execute_cmd(multihost, "userdel -rf foo2")
 
+    def test_bz1927195(self, multihost, backupsssdconf):
+        """
+        :title: sssd runs out of proxy child slots and
+          doesn't clear the counter for Active requests
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1927195
+        :id: 4c5b53e2-ec88-11ec-a41a-845cf3eff344
+        :steps:
+            1. Configure sssd with proxy
+            2. Try sssctl user-checks user@domain 10 times
+            3. Logs should not have error 'All available
+               child slots are full'
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
```

Comment:
4. Logs should have single count of error message caused by 11th user-check
5. Logs should have single count of error message from 11th user-check, as 12th user-check works

Reply:
DOne